src/fastfield/multivalued/reader.rs

```diff
@@ -191,6 +179,44 @@ impl<T: MonotonicallyMappableToU128> MultiValuedU128FastFieldReader<T> {
     }
 }
 
+impl<T: MonotonicallyMappableToU128> Column<T> for MultiValuedU128FastFieldReader<T> {
```

Comment:
```suggestion
// TODO having something that looks like MultiColumn trait. 
// See discussion in #1679
impl<T: MonotonicallyMappableToU128> Column<T> for MultiValuedU128FastFieldReader<T> {
```

Reply:
I added a comment. I'd really feel better if we can remove this... Is there a path where we just use the concrete type for the moment?